css/custom.css

```diff
@@ -177,3 +177,8 @@
     clear: left;
   }
 }
+
+.text-middle {
```

Comment:
This is not required. Bootstrap already has a class `text-center` which you can use straight away.